native/packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.android.js.snap

```diff
@@ -27,7 +27,7 @@ exports[`Android BpkButton should render correctly 1`] = `
           "margin": 10,
           "width": 300,
         },
-        undefined,
+        null,
```

Comment:
Nothing's jumping out at me that'd cause this - I'm guessing it's the result of a style that was being passed down as undefined to Android before due to the wrapper/container, but now isn't?

Reply:
Yeah I mean either way it's the same behaviour - I think it could be related to the theme 🤔